### ISSUE TITLE
Commands cannot be imported to Dragon 13

### ISSUE THREAD

**Issue by josephsl** [dictationbridge/dictationbridge-nvda#2]
Hi,

Raised by a user running Windows 10 64-bit: when trying to import commands to dragon 13, NVDA says that this cannot be done.

I asked this user for some diagnostics info in hopes that we can find out what's going on.

Thanks.

**Comment by josephsl** [comment]
Hi,

Another user raised a concern about this.

Technical: assuming that Dragon Naturally Speaking will be found in specific directories might be the root cause. Isn't there a way to find out by probing the registry instead?

Thanks.

**Comment by josephsl** [comment]
Hi,

To whoever will be writing a solution for this, I recommend the following:

1. Look up registry under HKLM\\Software\\Nuance\\Dragon Assistant 3 and raise an error if it's not there.
2. Look up the program files environment variable and see if Dragon is instlaled into the default directory. By default, it'll be found in sysdrive\\Program Files (X86)\\Nuance\\NaturallySpeakingNN where NN denotes version number (hint: use a regexp).
3. Copy the command files (this must be separated into a function of its own in order to allow MSR macros to be copied automatically).

Thanks.

**Comment by josephsl** [comment]
Hi,

Inaddition: if multiple Dragon versions are installed, ask the user which version one would like to use 9a separate issue).

Thanks.

**Comment by josephsl** [comment]
Hi,

Key found:

HKLM\\Software\WOW6432Node\\Nuance\\NaturallySpeaking15

Thanks.

**Comment by josephsl** [comment]
Hi,

Superseeded by issue #3 commits. Thanks.

**Comment by derekriemer** [comment]
Are you certain that key exists for dragon 13?
